specification.md

```diff
@@ -0,0 +1,92 @@
+# OpenTelemetry/Prometheus Compatibility Specification
+
+Status: [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+
+## Abstract
+
+OpenTelemetry is aiming to provide compatibility with
+Prometheus and OpenMetrics. This document explains the
+extent of the support. OpenTelemetry collector and libraries
+will align with the compatibility requirements defined in
+this spec.
+
+## Goals
+
+* OpenTelemetry collector can be used as a drop-in replacement
+  for Prometheus server to scrape and export metrics data.
+* OpenTelemetry collector should export OTLP-compatible Prometheus
+  time series to OpenTelemetry metrics exporters.
+* OpenTelemetry libraries should implement exporters to publish
+  Prometheus metrics.
+* To the OpenTelemetry collector, an OpenTelemetry target and
+  a Prometheus instrumented target is indistinguishable.
+
+## Differences and Limitations
+
+OpenTelemetry and Prometheus/OpenMetrics have different design
+goals and this reflects in the data model and implementation
+details. This section summarizes a few key differences.
+
+* **Pull vs push**: Prometheus is mainly designed for pull
+  whereas OpenTelemetry primarily is designed for push. This
+  difference causes how the state is maintained throughout the
+  collection including how it’s handled in the OpenTelemetry
+  collector.
+* **Cumulative vs delta**: OpenTelemetry supports delta temporality
+  whereas Prometheus always expects absolute/cumulative values. This
+  breaks some components that produce and communicate deltas where
```

Comment:
I'd like to be careful about what it means to "break" a component. The question, I guess, is what happens when Delta-temporality data arrives at a PRW or OpenMetrics exporter? That will indeed be an error, but that doesn't break Prometheus compatibility. The OTel data model is designed so that when used correctly (includes inside an SDK), Delta temporality can be translated to Cumulative temporality before one of these exporters sees the data.

Reply:
Do deltas carry sequence numbers or similar? How does the rebuild deal with lost/overlapping/delayed data?